### DISCUSSION TITLE
Create/edit modal example

### DISCUSSION THREAD

**Discussion by spitfire64** [Power-Components/livewire-powergrid#377] [Ideas]
## Feature Request

I like to see a working example of a modal form for editing/creating a powergrid item.

### Why is this needed?

I think that this I needed for people like me, that recently dived into livewire and found this fantastic project

### Suggested Solution(s)

I think that methods create() edit() and store() is needed as also methods related to opening/closing the modal. And of course the modal form itself.

A good starting point might be powergrid-demo that has the method Livewire/DishesTable/editDish() that for now only dumps the item that is clicked. In that method, the item should be loaded from the database and then handled over to the modal.
I think that an example that only makes the dish name editable would do. The main thing is to show how the edit/store modal should be implemented in Powergrid.

**Comment by dansysanalyst** [comment]
Hi @spitfire64,

That is a cool idea 😎

You are welcome to send a PR to the PowerGrid Demo repository.

**Reply by spitfire64** [reply]
OK, I promise that it will be in very small portions, will take time, and that I will need help :-)

**Comment by CrazyBoy49z** [comment]
All information here https://github.com/wire-elements/modal
